pdfgen/src/types/hierarchy/primitives/font.rs

```diff
@@ -30,23 +33,40 @@ impl Font {
         FONT,
         SUBTYPE,
         BASE_FONT,
+        NAME,
     }
 
-    /// Create a new Font object with the provided id, subtype and base_font.
-    pub fn new<S, B>(id: ObjId, subtype: S, base_font: B) -> Self
+    /// Create a new [`Font`] object with the provided id, subtype and base_font.
+    pub fn new<N, S, B>(name: N, id: ObjId, subtype: S, base_font: B) -> Self
     where
+        N: Into<Vec<u8>>,
```

Comment:
Why don't we use `OwnedName` or something similar here? I think we discussed it, can't remember what the final decision was.